tutorials/XGI in 15 minutes.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<p>comment on how many nodes and edges were removed?</p><p>also maybe before cleaning, get the number of isolates/singletons and see that the final count of the clean H is consistent?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>